src/plugins/data/public/ui/search_bar/search_bar.tsx

```diff
@@ -45,6 +46,8 @@ import { QueryEditorTopRow } from '../query_editor';
 import QueryBarTopRow from '../query_string_input/query_bar_top_row';
 import { SavedQueryMeta, SaveQueryForm } from '../saved_query_form';
 import { FilterOptions } from '../filter_bar/filter_options';
+import { SearchData } from '../../../../discover/public';
```

Comment:
Is the `SearchData` import being used anywhere?

Reply:
deleted